Automattic/jetpack
# Social Use dataviews in post share status

Fixes #

## Proposed changes:

*

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Enable Social sharing from Jetpack Settings > Sharing
* Connect some social accounts
* Create a post and publish it to share to some social media platforms
* Wait for sharing to complete in Jetpack sidebar in the editor
* Click on "View sharing history" in the Jetpack sidebar under "Share this post"
* Confirm that you see the share status modal same as on `trunk`
